src/js/test/index.test.js

```diff
@@ -9,28 +9,14 @@ describe('module', () => {
 });
 
 describe('setupHeader', () => {
-    it('converts nav checkbox to button with correct type attribute', () => {
-        // Arrange
-        TestUtils.setBodyHtml(`
-            <input data-nav-enhance />
-        `);
-
-        // Act
-        setupHeader();
-
-        // Assert
-        const html = TestUtils.getBodyHtml();
-        expect(html).toMatchSnapshot();
-    });
-
     it('adds `is-visible` class to nav container', () => {
         // Arrange
         TestUtils.setBodyHtml(`
-            <input data-nav-enhance />
+            <button data-nav-button></button>
             <div data-nav-container></div>
         `);
         setupHeader();
-        const button = document.querySelector('button[data-nav-enhance]');
+        const button = document.querySelector('[data-nav-enhance]');
```

Comment:
Should be `data-nav-button` now? (would think this test should be failing as the button wouldn't be clicked?)